src/js/modules/addToMyLibrary.js

```diff
@@ -0,0 +1,43 @@
+import Notiflix from 'notiflix';
+import { load, save } from './storage.js';
+
+const watchedMoviesKey = 'watchedMovies';
+const queuedMoviesKey = 'queuedMovies';
+
+const watched = load(watchedMoviesKey) || [];
+const queue = load(queuedMoviesKey) || [];
+
+const saveWatched = () => save(watchedMoviesKey, watched);
+const saveQueued = () => save(queuedMoviesKey, queue);
+
+export const setWatched = (id) => {
+    if (watched.includes(id)) {
+        Notiflix.Notify.warning('This movie is already in your watched list.');
```

Comment:
There will be problem with this lookup method e.g. when you look for 123 and there is 1234 but not 123 inside, it still gives you a match. Gotta resolve it myself later. I wrote the same comment to Robert and I reckon Mateusz before. If you have time any of you of course can fix it for me.